meson.build

```diff
@@ -15,37 +15,39 @@ cppcompiler = meson.get_compiler('cpp').get_argument_syntax()
 if not meson.is_subproject()
 
 # テストプロジェクト用の設定追加
+ldflags = []
 if cppcompiler == 'msvc'
     # VSプロジェクトに編集しうるファイルを追加する
     vs_files = ['include/rivet.hpp']
-    options = ['/std:c++latest', '/source-charset:utf-8', '/Zc:__cplusplus']
+    cxxflags = ['/std:c++latest', '/source-charset:utf-8', '/Zc:__cplusplus']
```

Comment:
While I'm here, I just happened to notice this pre-existing code...

Meson already passes the cplusplus option since Meson 0.63.0: https://mesonbuild.com/Release-notes-for-0-63-0.html#msvc-now-sets-the-__cplusplus-define-accurately

Meson historically passed `/utf-8` unless you specifically override the `/source-charset`, which means you don't need to override it to utf-8, Meson will do it for you.

Reply:
> Meson already passes the cplusplus option since Meson 0.63.0: https://mesonbuild.com/Release-notes-for-0-63-0.html#msvc-now-sets-the-__cplusplus-define-accurately

I didn't know this, thank you!

> Meson historically passed /utf-8 unless you specifically override the /source-charset, which means you don't need to override it to utf-8, Meson will do it for you.

I didn't know this either, but I don't think it was the case, at least in older versions (around 0.5x).  
Also, `/utf-8` also sets `/execution-charset:utf-8`, which is problematic in a Japanese runtime environment...